app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> str:

```

Comment:
Code Style: The `__init__` method should not return anything. Please remove the `-> str` from the method signature.